pkg/config_legacy/types/compute.go

```diff
@@ -30,21 +30,21 @@ type JobTimeoutConfig struct {
 	// check.
 	JobExecutionTimeoutClientIDBypassList []string `yaml:"JobExecutionTimeoutClientIDBypassList"`
 	// JobNegotiationTimeout default timeout value to hold a bid for a job
-	JobNegotiationTimeout Duration `yaml:"JobNegotiationTimeout"`
+	JobNegotiationTimeout Duration `yaml:"JobNegotiationTimeout" swaggertype:"primitive,integer"`
```

Comment:
not sure why we are trying to change config_legacy types. They shouldn't be in use except in the migration path

Reply:
Until and unless we get rid of them from the repo, they will be generated, as they are in path of our swagger generation.